### PR TITLE
DEV: First desktop design interface pass at experimental sidebar take 2.

### PR DESCRIPTION
**Pull request by tgxworld**
First approved in https://github.com/discourse/discourse/pull/16590 but was reverted as styling on an internal plugin broke.

### CODE REVIEW

File: app/assets/javascripts/discourse/app/templates/application.hbs [hunk 1]
```diff
@@ -8,26 +8,34 @@
...
-      {{/if}}
-      {{notification-consent-banner}}
-      {{pwa-install-banner}}
-      {{global-notice}}
-      {{create-topics-notice}}
-      {{plugin-outlet name="top-notices" connectorTagName="div" args=(hash currentPath=router._router.currentPath)}}
-    </div>
+  <div id="main-outlet-wrapper" class="wrap {{mainOutletWrapperClasses}}" role="main">
+    {{#if currentUser.experimental_sidebar_enabled}}
+      <Sidebar @shouldDisplay={{showSidebar}} />
```
Comment (nattsw) [inline app/assets/javascripts/discourse/app/templates/application.hbs:20]: Oooh nice

File: app/assets/javascripts/discourse/app/widgets/header.js [hunk 2]
```diff
@@ -400,7 +400,12 @@ export default createWidget("header", {
       return panels;
     };
 
-    let contentsAttrs = { contents, minimized: !!attrs.topic };
+    let contentsAttrs = {
```
Comment (nattsw) [inline app/assets/javascripts/discourse/app/widgets/header.js:403]:
```suggestion
    const contentsAttrs = {
```

**Comment by nattsw** [comment]
Are you seeing this @tgxworld ?

Screenshot 2022-05-05 at 2 11 15 PM

**Comment by tgxworld** [comment]
@nattsw Fixed. It looks like `.sidebar-wrapper` is a CSS class being used by the internal plugin too.

**Comment by nattsw** [comment]
@tgxworld if you force push I can't easily see the change you made 😄

**Review by nattsw — APPROVED** [review]

**Comment by nattsw** [comment]
🚀

**Comment by tgxworld** [comment]
Thank you @nattsw

**Pull request merged**